docs/includes/topic_sync-mode.rst

```diff
@@ -5,8 +5,8 @@ Sync mode
 
 * ``f5_sync_mode``: Defines the model by which policies configured on one BIG-IP® are shared with other BIG-IP®s.
 
-   * ``autosync``: configurations are synced automatically across all BIG-IP®s in a device cluster. [#fn1]_
-   * ``replication``: each device is configured separately.
+   * ``autosync``: uses BIG-IP® sync mode to sync LBaaS changes across all devices in a cluster. [#]_
```

Comment:
I was planning on taking out of the docs when it's actually removed from the config options. For now, I just have it marked as unsupported.